sindresorhus/refined-github
# Fix duplication and cropped hovercard in `user-local-time`

Fixes #3634 
Fixes #4345 

## Test URLs

Steps to reproduce the bug:
1. Clear the extension cache
2. Go to a page with a lot of usernames, e.g. https://github.com/sindresorhus/refined-github/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc
3. Hover over the various usernames until the bug appears (it may take a while)

For #4345: https://github.com/hexalellogram/iPhoneBatteryClient

## Screenshot

https://user-images.githubusercontent.com/46634000/120510278-8f014500-c3c9-11eb-8473-c11fe6a42449.mp4

For the comment in the CSS:
![image](https://user-images.githubusercontent.com/46634000/120511483-bf95ae80-c3ca-11eb-8a97-310b32ae96c0.png)
